backend/papersfeed/tests/tests_recommendation.py

```diff
@@ -37,6 +40,205 @@ def setUp(self):
             modification_date="2019-11-13",
         )
 
+        Paper.objects.create(
+            title="paper2",
+            language="English",
+            abstract="abstract1",
+            ISSN="1",
```

Comment:
[NOT IMPORTANT] You don’t have to give some fields like ISSN, eISSN, DOI, and dates.